modules/installation-complete-user-infra.adoc

```diff
@@ -130,6 +133,51 @@ command.
 +
 If the pod logs display, the Kubernetes API server can communicate with the
 cluster machines.
+ifdef::ibm-z[]
+. For an installation with FCP, additional steps are required to enable multipathing. 
```

Comment:
@bgilbert - This is the point of reference from the #multipath-47 Slack thread. 
Do you know if this addition should apply to the common UPI install docs as well? In other words, should we include these additional steps for installation with FCP to enable multipathing, or is it specific just to Z/Power?

Reply:
It sounds as though there's now a consensus in the Slack channel about how to handle this.  I'm not a multipath SME, actually, so I'll defer to the consensus.